spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -291,16 +260,10 @@ function onRecvPacket(packet: Packet) {
 function onAcknowledgePacket(
   packet: Packet,
   acknowledgement: bytes) {
-  switch (ack.type) {
-  case Type.REGISTER:
-    if (ack.code === 0) {
-      onAccountCreated(packet.sourcePort, packet.sourceChannel, ack.data)
-    }
-    return
-  }
-  case Type.RUNTX:
-    if (ack.code === 0) {
+    if (acknowledgement.result) {
       onTxSucceeded(packet.sourcePort, packet.sourceChannel, packet.data.data)
+      // Sets the interchainAccount address on the controller side for a given owner
+      setInterchainAccountAddress(sourcePort, result)
```

Comment:
This doesn't make sense. The result is not the interchain address anymore from my understanding. Should this be removed?

Reply:
As colin suggests here: https://github.com/cosmos/ibc/pull/567/files#r669637554

We want to return the address of the interchain account on packet sequence 1. So in this case result would include the address.

Reply:
https://github.com/cosmos/ibc/pull/567/files#diff-56983566230cc9f705648cb67111ac9a42e097e0de832ed165b6da08331cb299R249